src/oemof/solph/flows/_non_convex_flow_block.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
```

Comment:
This file here is difficult to review in details

Reply:
It should be just refactored in the sense that code has been grouped in functions.